comicScraper.py

```diff
@@ -109,6 +104,10 @@ def scrapeImageLinksFromIssue(url):
             imageUrl = extractImageUrlFromText(line)
             imageLinks.append(imageUrl)
 
+        if checkForCaptcha(line,url):
+            # Note: currently we are not actually solving the captcha so this will start an endless loop
+            return scrapeImageLinksFromIssue(url)    
```

Comment:
we should add a loop here that kills the script if the function gets called more than a couple of times, the `start an endless loop` part is worrying to me.